packages/medusa-plugin-auth/src/auth-strategies/facebook/store.ts

```diff
@@ -65,7 +69,23 @@ export class FacebookStoreStrategy extends PassportStrategy(FacebookStrategy, FA
 				.catch(() => void 0);
 
 			if (customer) {
-				if (!customer.metadata || !customer.metadata[CUSTOMER_METADATA_KEY]) {
+				// To prevent Legacy applications from not authenticating because only CUSTOMER_METADATA_KEY was set
```

Comment:
We can maybe add a todo to remove it in the next major?

Reply:
I thought we wanted to keep this moving forward to allow for backward compatibility with those currently using the authentication service?